src/components/avatar/avatar-group.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import Popover from 'components/popover'
+import Avatar from '.'
+import AvatarContext from './avatar-context'
+
+/**
+ * AvatarGroup Properties
+ */
+export type AvatarGroupProps = {
+  /** Render Avatars in AvatarGroup as children */
+  children: React.ReactNode
+  /** Maximum number of Avatars to show */
+  maxCount?: number
+  /** The placement of excess avatar Popover */
+  maxPopoverPlacement?: 'top' | 'bottom'
+  /** Customize style of excess avatar */
+  maxStyle?: React.CSSProperties
```

Comment:
If we are allowing the user to pass `style` prop, then we should also allow them to pass `className` prop as well. Add `maxClassName`. And its name can be converted to `maxStyle` to `excessAvatarStyle` and `excessAvatarClassName`.